gumroad/README.txt

```diff
@@ -2,30 +2,32 @@
 Contributors: karloscarweber, pderksen, nickyoung87, gumroad
 Tags: gumroad, gumroad product pages, gumroad overlay, gumroad embed, ecommerce, e-commerce, pdf, javascript, overlay, embed
 Requires at least: 3.9
-Tested up to: 4.9.6
-Stable tag: 2.1
+Tested up to: 5.7.1
```

Comment:
I have tested this plugin on a locally running latest version of WordPress (v5.7.1). Not sure if we could take this opportunity to fix https://github.com/gumroad/wp-gumroad/issues/41 somehow if it involves ticking a simple checkbox while uploading the updated ZIP archive of the plugin.

Reply:
According to https://wordpress.org/support/topic/this-plugin-hasnt-been-tested-with-the-latest-3-major-releases-of-wordpress/#post-9962705, maybe #41 will be automatically resolved with this change.